crates/next-dev/tests/integration/next/image/basic/index.js

```diff
@@ -0,0 +1,5 @@
+// Our integration tests are setup to use the project root's index.js,
+// but next-dev uses pages/index.js. Just paper over this by having both.
+
+export * from "./pages/index.js";
+export { default } from "./pages/index.js";
```

Comment:
The whole file only exists to make the test suite happy?

Reply:
Whoops, forgot to delete this. An earlier attempt at getting this to work required this, because of the assert that I removed.